src/app/Dashboard/AutomatedAnalysis/AutomatedAnalysisConfigForm.tsx

```diff
@@ -324,125 +271,83 @@ export const AutomatedAnalysisConfigForm: React.FC<AutomatedAnalysisConfigFormPr
-            onClick={handleSaveConfig}
-            isDisabled={isFormInvalid || isSaveLoading}
-            {...saveButtonLoadingProps}
-          >
-            {isSaveLoading ? 'Saving' : 'Save configuration'}
-          </Button>
-          {showHelperMessage && !isSaveLoading && (
-            <HelperText className={`${automatedAnalysisRecordingName}-config-save-helper`}>
-              <HelperTextItem variant="success">
-                <Text component={TextVariants.p}>Automated analysis recording configuration saved.</Text>
-                {!props.isSettingsForm && (
-                  <Text>
-                    You can also change this in the&nbsp;
-                    <Link to="/settings">
-                      <CogIcon /> Settings
-                    </Link>
-                    &nbsp;view.
-                  </Text>
-                )}
-              </HelperTextItem>
-            </HelperText>
-          )}
-          {templateType == 'TARGET' && (
-            <HelperText className={`${automatedAnalysisRecordingName}-config-save-template-warning-helper`}>
-              <HelperTextItem variant="warning">
-                <Text component={TextVariants.p}>
-                  WARNING: Setting a Target Template as a default template type configuration may not apply to all
-                  Target JVMs if they do not support them.
-                </Text>
-              </HelperTextItem>
-            </HelperText>
-          )}
-        </ActionGroup>
+        {templateType == 'TARGET' && (
+          <HelperText className={`${automatedAnalysisRecordingName}-config-save-template-warning-helper`}>
+            <HelperTextItem variant="warning">
+              <Text component={TextVariants.p}>{t('AutomatedAnalysisConfigForm.TEMPLATE_INVALID_WARNING')}</Text>
+            </HelperTextItem>
+          </HelperText>
+        )}
       </>
     ),
     [
+      t,
       templateName,
       templateType,
       templates,
-      isLoading,
-      isSaveLoading,
       selectedSpecifier,
       maxSize,
       maxSizeUnits,
       maxAge,
       maxAgeUnits,
-      isFormInvalid,
-      showHelperMessage,
-      createButtonLoadingProps,
       handleMaxSizeChange,
       handleMaxAgeChange,
       handleMaxSizeUnitChange,
       handleMaxAgeUnitChange,
-      handleSaveConfig,
-      handleSubmit,
       handleTemplateChange,
-      props.isSettingsForm,
-      saveButtonLoadingProps,
     ]
   );
 
   if (errorMessage != '') {
     return (
       <ErrorView
-        title={'Error displaying recording configuration settings'}
+        title={t('AutomatedAnalysisConfigForm.ERROR_TITLE')}
         message={errorMessage}
         retry={isAuthFail(errorMessage) ? authRetry : undefined}
       />
     );
   }
+  if (isLoading) {
+    return <LoadingView />;
+  }
   return (
     <>
-      {props.isSettingsForm ? (
-        formContent
-      ) : (
+      {useTitle ? (
         <Form>
```

Comment:
I think we will still need to `Form` to be common parent here. Just remove the FormSection if `useTitle` is false.